src/micro.js

```diff
@@ -8,7 +8,8 @@ var c = document.currentScript,
   // v is the rate of the current script element
   v = q.get('rum') || q.get('optel') || d.rate,
   // w is the rate of the current script element
-  w = { on: 1, off: 0, high: 10, low: 1e3 }[v] || 100,
+  t = { on: 1, off: 0, high: 10, low: 1e3 }[v],
+  w = t !== undefined ? t : 100,
```

Comment:
The simplest would have been this, but [canIuse](https://caniuse.com/?search=nullish+coalescing) says 93% users have the support of this operator

`w = { on: 1, off: 0, high: 10, low: 1e3 }[v] ?? 100`